.github/workflows/on_merge_to_master.yaml

```diff
@@ -21,6 +21,12 @@ jobs:
         with:
           ssh-private-key: ${{ secrets.MOONLITE_AWS_EC2_SSH_KEY }}
 
+      - name: Create .env file
```

Comment:
this might not work remotely. The `.env` file will be created but I think it's also needed to be copied over to `/apps/stable-diffusion` on the ec2 instance. If you look at the deployment step, we just pull the master from this repo.
https://github.com/Moonlite-Media/stable-diffusion-webui/blob/ecc31f611d295d0a7374aef760619c7659c49dfa/.github/workflows/on_merge_to_master.yaml#L52